tests/SecurityGroup.py

```diff
@@ -130,11 +143,16 @@ def verifyNonParticipants(self, transId):
         # verify each nonParticipant in the list has not advanced its lib to the publish block, since the block that would cause it to become finalized would
         # never have been forwarded to a nonParticipant
         for nonParticipant in self.nonParticipants:
+            if nonParticipant.pid is None:
+                continue
             nonParticipantPostLIB = nonParticipant.getBlockNum(blockType=BlockType.lib)
             assert nonParticipantPostLIB < publishBlock, "Participants not in security group should not have advanced LIB to {}, but it has advanced to {}".format(publishBlock, nonParticipantPostLIB)
             nonParticipantHead = nonParticipant.getBlockNum()
             assert nonParticipantHead < producerHead, "Participants (that are not producers themselves) should not advance head to {}, but it has advanced to {}".format(producerHead, nonParticipantHead)
 
+    def getLatestPublishTransId(self):
```

Comment:
whoops, fixed that. It was so that if the caller didn't want to call verifySecurityGroup, and just was calling one of the functions that calls, it can get the default value.